### PR TITLE
Add marker UI clustering

### PR DESCRIPTION
**Pull request by james-woo**

### CODE REVIEW

File: DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿Additions allow you to add arbitrary C# to the generated classes
```
Comment (ToucheSir) [inline DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt]: Is it possible to omit this whole folder from the commit? It looks like we're including the whole project source or Xamarin.Android.Maps.Utils with this.

Comment (ToucheSir) [inline DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt]: Is there not a nuget package for it?

Comment (james-woo) [inline DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt]: Alright, removed project and added DLL

Comment (ToucheSir) [inline DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt]: Cool, that works too.

**Pull request merged**